src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -342,7 +342,9 @@ CHIP_ERROR ScenesServer::Init()
 
 void ScenesServer::Shutdown()
 {
+    chip::Server::GetInstance().GetFabricTable().RemoveFabricDelegate(&gFabricDelegate);
     chip::app::CommandHandlerInterfaceRegistry::Instance().UnregisterCommandHandler(this);
+    chip::app::AttributeAccessInterfaceRegistry::Instance().Unregister(this);
```

Comment:
```suggestion
    Server::GetInstance().GetFabricTable().RemoveFabricDelegate(&gFabricDelegate);
    CommandHandlerInterfaceRegistry::Instance().UnregisterCommandHandler(this);
    AttributeAccessInterfaceRegistry::Instance().Unregister(this);
```

Would be nice to remove the extra namespace prefixes in the Init method too...